config/config.yaml

```diff
@@ -1,6 +1,17 @@
-project_code: "sage-bionetworks"
-profile: {{ var.profile | default("default") }}
-region: {{ var.region | default("us-east-1") }}
-template_bucket_name: "bootstrap-awss3cloudformationbucket-114n2ojlbvj21"
-template_key_prefix: {{ environment_variable.TRAVIS_BRANCH | default("testing") }}
-admincentral_cf_bucket: "bootstrap-awss3cloudformationbucket-19qromfd235z9"
+project_code: 'sage-bionetworks'
+profile: {{ var.profile | default('default') }}
+region: {{ var.region | default('us-east-1') }}
+docker_image_tag: {{ environment_variable.DOCKER_IMAGE_TAG | default('staging-latest') }}
+domain_name: {{ environment_variable.DOMAIN_NAME | default('cri-iatlas.org') }}
```

Comment:
The DOMAIN_NAME and similar environment variables are currently set in GitLab to use GenUI domains instead of the `cri-iatlas.org` domain.